### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by yyyc514**
Attempt to answer question early and avoid creating of "please add language xyz" style issues.

### CODE REVIEW

**Comment by yyyc514** [comment]
@egor-rogov @allejo 

The middle portion is also the language I would like to add the `language-requests.rst` (replacing what it currently says) once we agree on it.

**Comment by egor-rogov** [comment]
Makes sense. There are a lot of such requests.

File: .github/ISSUE_TEMPLATE/language-request.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+about: I really wish Highlight.js could highlight ...
+title: PLEASE read the below carefully.
+labels: ''
+assignees: ''
+
+---
+
+First let us say that we'd also love it if Highlight.js could highlight whichever language you're about to request support for!  And there is a chance you can help make that happen! But first...
+
+...PLEASE READ THE FOLLOWING...
```
Comment (egor-rogov) [inline .github/ISSUE_TEMPLATE/language-request.md:12]: I personally don't think that PLEASE READ banners really help, and there are two of them (another one in the title). I'd remove this line, but I understand it's subjective. Feel free to ignore.

File: .github/ISSUE_TEMPLATE/language-request.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    Highlight.js does not have a fundamental plan for implementing new languages
+    - i.e., the core team doesn't usually develop new languages. The core team
+    instead focuses on parser development, bugs, and supporting the existing
+    languages. Core also currently does not have time to review, merge and
```
Comment (allejo) [inline .github/ISSUE_TEMPLATE/language-request.md:17]:
```suggestion
    languages. They also currently does not have time to review, merge and
```

**Pull request merged**